core/components/ui/pagination/pagination.tsx

```diff
@@ -45,25 +33,21 @@ const Pagination = ({
   }
 
   return (
-    <nav
-      aria-label="Pagination"
-      className={cn('my-6 text-center text-primary', className)}
-      {...props}
-    >
+    <nav aria-label="Pagination" className={cn('my-6 text-center text-primary', className)}>
       {hasPreviousPage ? (
-        <Link className="inline-block" href={`${pathname}?${beforeSearchParams.toString()}`}>
-          <span className="sr-only">{prevLabel}</span>
+        <CustomLink className="inline-block" href={`${pathname}?${beforeSearchParams.toString()}`}>
+          <span className="sr-only">Previous</span>
```

Comment:
Do we not want to take in labels for pagination translations anymore? Not against it, just curious

Reply:
I think we need to implement the translation directly on the components, probably in a different PR.